### ISSUE TITLE
Possible rendering glitches in Chrome/Safari when using ImageFit.cover

### ISSUE THREAD

**Issue by BrendanMcK** [OfficeDev/office-ui-fabric-react#6020]
- __Package version(s)__:  6.49.2
- __Browser and OS versions__: Chrome (Win+Mac) and Safari
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No): Unsure what fix is at this stage.
 
Requested priority: (Blocking, High, Normal, Low): Normal?
 
Products/sites affected: (if applicable): Unknown
 
#### Describe the issue:
 
We currently have a Tile component that we're working on, based on Fabric's Image. We noticed that when using this with a border, and when using ImageFit.cover, within a responsive parent element thats sizing to some % of available space, we'd see a white glitch line between the image and the border:

![repro](https://user-images.githubusercontent.com/3459020/44177809-60910e80-a0a4-11e8-8fd1-cc168b6e7a63.png)

This is a grab from a Retina-based laptop - but the glitch is visible on regular DPI screens also. It's mostly prominent along the right edge, but if you look at the pixels, there's a thinner glitch along the left edge also.

This seems to be a Webkit related issue - only Chrome and Safari appear to be affected; FF and Edge are fine here, with nice solid lines.

Here's a minimal repro using Fabric: https://codepen.io/brendanmckeon/pen/RBzzYY

And here's a minimal repro using pure HTML+CSS: https://codepen.io/brendanmckeon/pen/OoPvdO

In both these cases, a yellow glitch can be seen between the black image and the red 1px border:

![image](https://user-images.githubusercontent.com/1754373/44440454-fc2aef00-a57c-11e8-8f9e-82de83cc1ba8.png)

My guess is that this glitch is triggered by how image centering is handled here: Fabric is using `top: 50%` and `left: 50%` balanced out with a `transform: translate(-50%, -50%)` to do this. However, the top/left part of this happen during layout, while transform takes place at a different part of the rendering pipeline, and it seems that in WebKit/Chrome/Safari, they don't cancel each other out exactly (perhaps with how subpixel units or antialiasing is handled), so we get this glitch. Likely also significant is that this repros when the image is assigned 50% of a parent container that's not a clean even number of pixels.

Perhaps ideal is to avoid mixing layout vs composition transforms - are there other centering techniques available that don't involve transform?

Reporting as this might affect other customers - FYI @swese44

**Comment by natalieethell** [comment]
Hey @BrendanMcK thanks for catching this! After trying a couple different approaches, I think `display: inline-table` might help us out here. Just testing it out in this CodePen, I'm not about to repro the rendering glitch: https://codepen.io/naethell/pen/LJNMmw. It seems to force rendering of exact pixel widths/heights. I still need to check for regressions, but I just wanted to post an update here!

**Comment by natalieethell** [comment]
Actually, that doesn't seem to work in all cases -- and actually doesn't work at all in Firefox. We'll keep looking.

**Comment by natalieethell** [comment]
Alright, so here's what we've found. We tried a couple different approaches, but the real issue lies in the subpixel rendering. This is a CSS rounding bug that you'll find regardless of what's rendered inside of the wrapper, as @micahgodbolt has shown in this CodePen (which also doesn't have any Fabric components): https://codepen.io/micahgodbolt/pen/OoNGvb.

So, since this is not a Fabric-specific bug, I'll close this. 

If we think of a holistic solution in the future, we'd be happy to share and let us know if you have any questions or find another solution!
